### PR TITLE
Instrument sidekiq

### PR DESCRIPTION
**Pull request by jkassemi**
Includes `sidekiq-instrument` (https://github.com/enova/sidekiq-instrument) to provide the prometheus statsd exporter details about sidekiq's operation.

Runs a scheduled job every minute to collect details from sidekiq for overall usage, and installs sidekiq middleware to track job queue and processing times.

### CODE REVIEW

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -2,3 +2,7 @@ CreateDailySpoolFiles:
   cron: "0 3 * * * America/New_York"
   class: EducationForm::CreateDailySpoolFiles
   description: "Generate a daily set of spool files and transmit them over SFTP to a backend system"
+
+SidekiqStatsJob:
+  every: 1m
```
Comment (markolson) [inline config/sidekiq_scheduler.yml]: Can you test locally and ensure that this won't be run multiple times per minute - one per Scheduler instance? https://github.com/moove-it/sidekiq-scheduler/commit/ee317fd427bf600ba2562f9ae3cc64b9880d0e68 makes me think that only `cron` and `at` type jobs are de-duplicated.

Comment (jkassemi) [inline config/sidekiq_scheduler.yml]: Nice catch - yes, it was running multiple times per minute. Tested locally after update to use `cron` option and job is only executed once across all workers.

**Comment by aub** [comment]
@jkassemi Any reason to keep the blank spec file in there? LGTM otherwise.

**Comment by markolson** [comment]
While this is unmerged, maybe add a comment saying to only use `cron` or `at`?

**Comment by jkassemi** [comment]
The spec loads the worker, which gives us the required code coverage (ensures that there are no syntax issues, required constants defined, etc).

**Pull request merged**